### PR TITLE
Added some doc to system_metric_collector

### PR DESCRIPTION
**Pull request by ahcorde**
Added some doc to system_metric_collector

### CODE REVIEW

File: system_metrics_collector/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# System metric collector
+
+## Parameters
+
+There are two parameters defined:
+
+  - **measurement_period**:
+```sh
```
Comment (piraka9011) [inline system_metrics_collector/README.md]: consider using `yaml` better readability

File: system_metrics_collector/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  Constraints:
+    Read only: true
+    Min value: 1
+    Max value: 9223372036854775807
+    Step: 1
+```
+
+## Usage:
+
+You can run the following command to generate CPU and Memory statistics. for both, all the system and the node running:
```
Comment (piraka9011) [inline system_metrics_collector/README.md]: Remove the second sentence, you only need the first

File: system_metrics_collector/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+```
+ros2 run system_metrics_collector main
+```
+
+Change `publish_period` or `measurement_period` using `--ros-args`:
```
Comment (piraka9011) [inline system_metrics_collector/README.md]: `publish_period` or `measurement_period`  can be modified using...

File: system_metrics_collector/README.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    Max value: 9223372036854775807
+    Step: 1
+```
+
+## Usage:
+
+You can run the following command to generate CPU and Memory statistics. for both, all the system and the node running:
+
+```
+ros2 run system_metrics_collector main
```
Comment (dabonnie) [inline system_metrics_collector/README.md]: This usage isn't quite correct. At some point in the next 2 weeks, we will be moving this main to an example directory. The preferred way to launch the system CPU and system memory collector will be to use their entry points (standalone mains) or the launch file. Per process CPU and memory will use composition.

Comment (thomas-moulard) [reply system_metrics_collector/README.md]: @dabonnie Is this the correct usage now? If yes, I think we can merge as it is and the refactoring PR should update the doc accordingly at this time?

Comment (dabonnie) [reply system_metrics_collector/README.md]: @thomas-moulard sounds good, why I didn't submit as a non-blocking review ;-)

**Comment by dabonnie** [comment]
Merging as build passes but codecov is broken.

**Pull request merged**